packages/edit-navigation/src/store/transform.js

```diff
@@ -31,12 +163,10 @@ export default function menuItemsToBlocks( menuItems ) {
  * @return {Object} Object containing innerBlocks and mapping.
  */
 function mapMenuItemsToBlocks( menuItems ) {
-	let mapping = {};
-
 	// The menuItem should be in menu_order sort order.
 	const sortedItems = sortBy( menuItems, 'menu_order' );
 
-	const innerBlocks = sortedItems.map( ( menuItem ) => {
+	const blocks = sortedItems.map( ( menuItem ) => {
```

Comment:
Noting for later: This doesn't match nested blocks with their related menu item ids – let's figure it out in another PR.

Reply:
👍 Sounds good. Would be great to see an issue for it.

Reply:
Tracked in https://github.com/WordPress/gutenberg/issues/35047